tests/integration/targets/ec2_eip/tasks/main.yml

```diff
@@ -0,0 +1,1442 @@
+- name: Integration testing for ec2_eip
+  module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region }}'
+    ec2_eip:
```

Comment:
Same thing, it seems like this needs to be `amazon.aws.ec2.eip` now.

Reply:
Ah, done, thanks!